samples/CGCatalog/CGCatalog/CGAddPathToPointView.m

```diff
@@ -0,0 +1,174 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "CGAddPathToPointView.h"
+
+@interface CGAddPathToPointView () <UITableViewDelegate, UITableViewDataSource>
+
+@property (nonatomic, strong) NSMutableArray* functions;
+@property (nonatomic, strong) UIImageView* testViewDisplayView;
+@property (nonatomic, strong) UITableView* consoleView;
+@property (nonatomic, strong) UISlider* slider;
+
+@end
+
+@implementation CGAddPathToPointView
+
+- (void)loadView {
+    [super loadView];
+
+    self.testViewDisplayView =
+        [[UIImageView alloc] initWithFrame:CGRectMake(0.0f, 0.0f, self.view.bounds.size.width, self.view.bounds.size.height / 2.0f)];
+    [self.testViewDisplayView setBackgroundColor:[UIColor redColor]];
+    [self.view addSubview:self.testViewDisplayView];
+
+    self.consoleView = [[UITableView alloc] initWithFrame:CGRectMake(0.0f,
+                                                                     self.view.bounds.size.height / 2.0f,
+                                                                     self.view.bounds.size.width,
+                                                                     self.view.bounds.size.height / 2.0f)];
+    [self.consoleView setRowHeight:500.0f];
+    [self.consoleView setEstimatedRowHeight:100];
+    [self.consoleView setDelegate:self];
+    [self.consoleView setDataSource:self];
+    [self.view addSubview:self.consoleView];
+}
+
+/*- (void)drawPaths:(CGAffineTransform)transform {
```

Comment:
This file should have been deleted. Will remove.

---
In reply to: [80592524](https://github.com/Microsoft/WinObjC/pull/1033#discussion_r80592524) [](ancestors = 80592524)